src/commands/install-node.yml

```diff
@@ -59,11 +59,36 @@ steps:
         - run:
             name: Install and use node version
             command: |
-              # if("<<parameters.version>>" -eq "latest") {
-              #   choco install nodejs
-              # } else {
-              #   $r = Invoke-WebRequest https://nodejs.org/download/release/index.json | ConvertFrom-Json
-              #   $latest_of_version = ($r | where { $_.version -like "v<<parameters.version>>*" })[0].version.substring(1)
-              #   choco install nodejs --version $latest_of_version --allow-downgrade
-              # }
+              $nodeIndex = "https://nodejs.org/download/release/index.json"
+              $registry = (Invoke-WebRequest $nodeIndex).content
+
+              if ("<<parameters.version>>" -eq "latest") {
+                $latest = $registry | jq -c -r '.[0].version | ltrimstr(\"v\")'
+
+                nvm install $latest
+                nvm use $latest
+              } elseif ("<<parameters.version>>" -eq "lts") {
+                $lts = $registry |
+                    jq -c -r 'first(.[] | select(.lts != false)).version | ltrimstr(\"v\")'
+
+                nvm install $lts
+                nvm use $lts
+              } elseif ("<<parameters.version>>" -eq "maintenance") {
+                $active = $registry | jq -c -r 'first(.[] | select(.lts != false)).lts'
+                $maintenance = $registry |
+                    jq -c -r --arg active $active `
+                    'first(.[] | select(.lts != $active and .lts != false)).version | ltrimstr(\"v\")'
+
+                nvm install $maintenance
+                nvm use $maintenance
+              } else {
+                nvm install "<<parameters.version>>"
+                nvm use "<<parameters.version>>"
+              }
+
+              Start-Sleep -Seconds 1
```

Comment:
We have to wait 1 sec to be able to call node, see: https://app.circleci.com/pipelines/github/salesforcecli/toolbelt/663/workflows/d5807740-12e7-46ba-a593-324e8f751f4f/jobs/3631?invite=true#step-101-11

related:
https://github.com/coreybutler/nvm-windows/issues/650
https://github.com/coreybutler/nvm-windows/issues/324

Reply:
huh.  Any possibility 1 sec won't be enough?  People in those 2 github issues were using 5 and 2.  

Even if it's 1/100 times that's going to be annoying for the NUTs.

Reply:
If you feel pretty good about 1 we can start there and see.

Reply:
Is there a way to poll for it (while or try/catch, etc?)

Reply:
Considering we're talking about os level calls here I think 1 second is plenty, but wouldn't hurt to do 2 just in case.

As far as polling or try/catching we'd have to do something like test the output of `$(node --version)` until it gives us the version we're expecting (or at least doesn't tell us that `node` doesn't exist), but if for some reason the `nvm install`/`use` steps fail then it'll become an infinite loop.

Reply:
Yes, agree that 2 sec would be better.
I tried multiple times with 1 sec locally and thought it would be safe but haven't done the same testing on circle ([only tried with this one](https://app.circleci.com/pipelines/github/salesforcecli/toolbelt/664/workflows/f8ef7f57-2526-4c68-8870-5b067736e072/jobs/3640)).